_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+  channel.
+
+  What Fournier proposed this week was only creating a single commitment
+  transaction that would be shared between
+  Alice and Bob.  When creating the transaction, Alice creates two
+  private keys she'll need to spend her output and gives Bob the
+  corresponding public keys.  Bob then gives Alice a [signature
+  adaptor][topic adaptor signatures] for his half of the 2-of-2 multisig spend in the commitment
+  transaction; the adaptor can only be transformed into Bob's signature
+  if that signature will reveal one of Alice's private keys to Bob.  The
+  other private key Alice created is the revocation private key, which
+  she'll give to Bob if she wants to revoke this channel state in favor
+  of a later state.  Similarly, Bob generates his own keys and receives
+  a signature adaptor from Alice.
+
+  In this way, even though the commitment transaction is the same for
+  both parties, there will be a difference in the signatures (called
+  *witnesses*) each of them uses to unilaterally close the channel.
+  The difference in the signatures will give the party who didn't close
+  the channel a chance to create and broadcast a penalty transaction if
+  the channel was closed in an old state.  Otherwise, if the channel was
+  closed in its latest state, both parties get their money, just like in
+  the current LN.
+
+  The proposal also suggests using basically the same scheme to
+  transition from Hash Time Lock Contracts (HTLCs) to Point Time Lock
+  Contracts ([PTLCs][news92 ptlcs]), which is already a goal several LN contributors
+  have voiced support for in the past.
+
+  Advantages of this approach are that it might require less private
+  data storage (though this is [debated][zmn reply]), it makes unilateral closes
+  a bit smaller onchain, and it describes a path to using PTLCs that
+  will provide LN users with additional privacy against routing
+  node surveilance as well as bring the protocol a little closer to what it may
+  look like when it can take advantage of the benefits of [schnorr
+  signatures][topic schnorr signatures].
+
+- **Default signet discussion:** Michael Folkson [posted][default signet post] to the
+  Bitcoin-Dev mailing list a question about whether there should be a
+  default [signet][topic signet] similar to Bitcoin's existing default testnet.
+  Signets are test networks where valid blocks must be
+  signed by one or more trusted keys; this allows them to be more
+  predictable for testing than fully decentralized test networks that
+  are often subject to vandalism.  [BIP325][] describes the signet
+  protocol, which anyone can use to create their own signet, but there
+  is a [desire][bitcoin core default signet] to include a default signet into
+  Bitcoin Core (and possibly other software) that
+  users will be able to enable with a single configuration option.
+  Folkson's first question is whether anyone has any objection to this.
```

Comment:
ISTM s/is/was/ here in keeping with the tense used in this paragraph and the next one